CrabRobotLibrary.py

```diff
@@ -14,6 +14,8 @@ class Robot:
 	brick = EV3Brick()
 	left_wheel = Motor(Port.A)
 	right_wheel = Motor(Port.D)
+	color_sensor_left = ColorSensor(Port.S1)
+	color_sensor_right = ColorSensor(Port.S4)
```

Comment:
It looks like Zac added these same lines, so there is a conflict that needs to be resolved.